src/components/CardItem.js

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from "react";
+import { getFolder } from "./Api";
+import noImage from "../assets/noimage.jpeg";
+
+function getTimeAgo(createdAt) {
+  const now = new Date();
+  const createdDate = new Date(createdAt);
+
+  const timeDiff = now - createdDate;
+  const minutes = Math.floor(timeDiff / (1000 * 60));
+  const hours = Math.floor(timeDiff / (1000 * 60 * 60));
+  const days = Math.floor(timeDiff / (1000 * 60 * 60 * 24));
+  const months = Math.floor(timeDiff / (1000 * 60 * 60 * 24 * 30));
+  const years = Math.floor(timeDiff / (1000 * 60 * 60 * 24 * 365));
+
+  if (minutes < 2) {
+    return "1 minute ago";
+  } else if (minutes <= 59) {
+    return `${minutes} minutes ago`;
+  } else if (hours < 1) {
+    return "1 hour ago";
+  } else if (hours <= 23) {
+    return `${hours} hours ago`;
+  } else if (days < 1) {
+    return "1 day ago";
+  } else if (days <= 30) {
+    return `${days} days ago`;
+  } else if (months < 1) {
+    return "1 month ago";
+  } else if (months <= 11) {
+    return `${months} months ago`;
+  } else if (years < 1) {
+    return "1 year ago";
+  } else {
+    const roundedYears = Math.floor(months / 12);
+    return `${roundedYears} years ago`;
+  }
+}
+
+function CardItem() {
+  const [cardData, setCardData] = useState();
```

Comment:
- 데이터의 형태에 맞게 초기값을 넣어두시는 게 좋습니다.
- 배열 데이터인 만큼, 변수명도 그에 맞게 지어주시는 게 좋습니다.
```suggestion
  const [cardList, setCardList] = useState([]);
```